tools/PrepareRelease/Program.cs

```diff
@@ -11,20 +14,28 @@ public class Program
 
         public static void Main(string[] args)
         {
-            if (JobShouldRun(Integrations, args))
-            {
-                Console.WriteLine("--------------- Integrations Job Started ---------------");
-                GenerateIntegrationDefinitions.Run();
-                Console.WriteLine("--------------- Integrations Job Complete ---------------");
-            }
-
             if (JobShouldRun(Versions, args))
             {
                 Console.WriteLine("--------------- Versions Job Started ---------------");
                 SetAllVersions.Run();
                 Console.WriteLine("--------------- Versions Job Complete ---------------");
             }
 
+            var solutionDir = EnvironmentTools.GetSolutionDirectory();
+            Environment.SetEnvironmentVariable("SOLUTION_DIR", solutionDir);
+            var tracerHomeOutput = Path.Combine(solutionDir, "tools", "PrepareRelease", "bin", "tracer-home");
+            Environment.SetEnvironmentVariable("TRACER_HOME_OUTPUT_DIR", tracerHomeOutput);
+
+            var publishBatch = Path.Combine(solutionDir, "tools", "PrepareRelease", "publish-all.bat");
+            ExecuteCommand(publishBatch);
```

Comment:
Can you make the publish part a "Job" like versions and integrations?

Reply:
The plan was to have it be a pre-req for the integrations and MSI steps below.

Reply:
I'll nest it within the MSI section for now, as that's the only place it's really used.